src/Connection/ConnectionInterface.php

```diff
@@ -78,4 +79,14 @@ public function getTableSchema(string $name, $refresh = false): ?TableSchema;
      * @return void
      */
     public function setEnableSlaves(bool $value): void;
+
+    public function isLoggingEnabled(): bool;
+
+    public function isProfilingEnabled(): bool;
```

Comment:
I think it would be better named:

```php
public function isProfiling(): bool;
```

Reply:
We need to think about this with the rest of the names in mind.